### DISCUSSION TITLE
Failed to upload 4GB file with Premium Account

### DISCUSSION THREAD

**Discussion by ghost** [anasty17/mirror-leech-telegram-bot#1002] [Q&A]
I tried to leech a 2.5GB file which under the 4GB limit but it keep splitting the file to 2GB parts. I've added a USER_SESSION_STRING but it doesn't work. How to fix that issue

![Screenshot 2022-08-29 043551](https://user-images.githubusercontent.com/105369854/187098983-b5597ab1-96c4-4fa7-a95d-ac2fe94fc214.png)

**Comment by anasty17** [comment]
Send full logs!

**Reply by ghost** [reply]
full log
![Capture 1](https://user-images.githubusercontent.com/105369854/187099627-3a905dfc-b13e-4c18-8e03-23c7cc0cab7c.JPG)

**Reply by anasty17** [reply]
Send link

**Reply by anasty17** [reply]
It seems ur account not premium !

**Comment by anasty17** [comment]
/eval from bot import app
print(app.me.is_premium)


Do this in ur bot

**Reply by ghost** [reply]
Is this right
![Capture 2](https://user-images.githubusercontent.com/105369854/187102910-e35c4fcb-9432-428f-bb73-fe6984b56c67.JPG)
![Capture 4](https://user-images.githubusercontent.com/105369854/187102915-8dc13e89-87d5-48b1-9d00-323195ee5944.JPG)

**Reply by anasty17** [reply]
Bruh!
Write this cmd while using the bot, don't touch the code

**Reply by anasty17** [reply]
![Screenshot_20220829-040301_Plus](https://user-images.githubusercontent.com/77075674/187103566-47b9f732-773b-482e-bc4f-1d366596ed41.jpg)

**Reply by ghost** [reply]
sorry about that

**Reply by ghost** [reply]
here what I got
![Capture 3](https://user-images.githubusercontent.com/105369854/187105335-762267c9-5ca0-4767-89f2-d09b6c19314b.JPG)

**Reply by anasty17** [reply]
Ur account is not premium

**Reply by ghost** [reply]
That's impossible. 
![photo_2022-08-29_07-13-36](https://user-images.githubusercontent.com/105369854/187106634-02c0d4f2-5b8e-47ae-acb3-24b47769e33b.jpg)

**Reply by anasty17** [reply]
Then generate new session string. Ur bot say its not!

**Reply by ghost** [reply]
That problem is finished. but now getting another error
![Capture 6](https://user-images.githubusercontent.com/105369854/187112455-0ef3f515-6add-4881-b407-50d36bee0edf.JPG)

**Reply by anasty17** [reply]
Yeah simce u didn't read readme! I wrote in readme that u should use supergroup when u r using session string for uploading!
